Guides.md

```diff
@@ -3,9 +3,19 @@ layout: page
 title: Guides
 permalink: /guides/
 ---
-
 This page contains a number of short guides for working remotely with the files and applications you have accessible at CYD.
 
+
+## Accessing IDA and Lysator from CYD
+
+CYD-poolens' computers has the Application ThinLinc installed. ThinLinc is required in order to access the IDA systems from a remote location. This enables CYD-poolen users to work remotely with IDA or Lysator systems. 
```

Comment:
Nope - mig veterligen har Ida tagit bort SSH-noden.


> 5 juni 2016 kl. 00:52 skrev Rovanion Luckey <notifications@github.com>:
> 
> In Guides.md:
> 
> >  This page contains a number of short guides for working remotely with the files and applications you have accessible at CYD.
> >  
> > +
> > +## Accessing IDA and Lysator from CYD
> > +
> > +CYD-poolens' computers has the Application ThinLinc installed. ThinLinc is required in order to access the IDA systems from a remote location. This enables CYD-poolen users to work remotely with IDA or Lysator systems. 
> Inte strikt sant, man kan använda SSH också.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>